py3status/modules/wifi.py

```diff
@@ -63,13 +64,26 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = 'W: [{bitrate} {signal_percent} {ssid}|\?show down]'
     round_bitrate = True
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
 
+    class Meta:
+        deprecated = {
+            'remove': [
+                {
+                    'param': 'format_down',
+                    'msg': 'obsolete parameter use formatter in format',
+                },
+                {
+                    'param': 'format_up',
+                    'msg': 'obsolete parameter use formatter in format',
+                },
+            ],
+        }
```

Comment:
I think we can do better than this.

It would need a bit more code but we should be able to convert something like this imho
```
wifi {
    format_down = 'Wifi: no connection'
    format_up = 'Wifi: {ip}'
}
```